### PR TITLE
New domain names are not supported by devkit

### PR DESCRIPTION
**Pull request by k-irishina**
Related to/goes together with: https://github.com/techcloud0/bankaxept-epayment-platform/pull/1986

### CODE REVIEW

**Comment by k-irishina** [comment]
Tests are expected to be failing, I guess, as the address is serving traefik default cert right now, so client calls fail with PKIX path building exceptions due to a self-signed cert.

**Review by adriantimberlid — DISMISSED** [review]
LGTM! Let's wait until merge in epp main repo. ⚡ 

Edit: Also a conflict that needs to be resolved first. 😄

**Review by adriantimberlid — APPROVED** [review]

**Pull request merged**
